data/event-schemas/schema/m.room.power_levels.yaml

```diff
@@ -5,6 +5,11 @@ description: |-
   This event specifies the minimum level a user must have in order to perform a
   certain action. It also specifies the levels of each user in the room.
 
+  The range of power levels is specified by the matrix-specific Canonical JSON
+  specification (https://spec.matrix.org/v1.3/appendices/#canonical-json), which
```

Comment:
```suggestion
  specification (/appendices/#canonical-json), which
```

Reply:
also we should just use a markdown link over "Canonical JSON", as we do elsewhere. It doesn't need the "matrix-specific" qualifier.

Reply:
Good point, fixed.